packages/satori-js/api.gen.ts

```diff
@@ -20,6 +20,10 @@ export interface ApiAuthenticateRefreshRequest {
 
 /**  */
 export interface ApiAuthenticateRequest {
+  //Optional custom properties to update with this call. If not set, properties are left as they are on the server.
+  custom?: Record<string, string>;
```

Comment:
Could you add these as optional parameters to the `authenticate` function? We use the generated diffs as reminders to update the interface.

Reply:
Since this belongs to other endpoint, I will actually remove this change set for now